static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -120,6 +121,19 @@ const loadAnalytics = (): void => {
     }
 };
 
+const loadGoogleAnalytics = (): void => {
+    const handleGoogleAnalytics = (gaHasConsent: boolean): void => {
+        if (gaHasConsent && !config.get('page.gaIsInitalised')) {
+            window.guardian.config.googleAnalytics.initialiseGa()
```

Comment:
I (rightly) get a validation check error here:

`error    use the 'config' module instead of window.guardian.config`

but I'm not sure what to do about it. 

neither `config.get('googleAnalytics.initialiseGa')` or `config.get('googleAnalytics.initialiseGa()')` call the function...

Reply:
This is moot if you don't put it in `guardian.config` but this probably needs to be `config.get('googleAnalytics.initialiseGa')()` with additional handling for if `config.get` returns null. For that I think you can do `config.get('googleAnalytics.initialiseGa', () => {})()` or similar (second argument to `get` is for a default value)

Reply:
Excellent, I've moved it from config now, but useful for next time.